pkg/ingresscontroller/ingresscontroller.go

```diff
@@ -0,0 +1,139 @@
+package ingresscontroller
```

Comment:
Please rename package to ingress and directory ./pkg/ingress. Under ingress we could have several resources like:
1. Ingress
2. Ingresscontroller 
and more